src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -174,7 +174,9 @@ void ResourceAllocationManagerImpl::ProcessApplicationPolicyUpdate() {
       if (rc_extention) {
         rc_extention->UnsubscribeFromInteriorVehicleData(*module);
       }
-      SendOnRCStatusNotification();
+      SendOnRCStatusNotifications(
```

Comment:
this is inside a `for` loop. My concern is if there is a policy update that disallows 2 or more moduleTypes from an application, will this trigger 2 or more notifications to the changing app and HMI. It is preferred to trigger one notification to the app/hmi per one policy update no matter how many moduletTypes are disallowed.

Reply:
I'd like to move SendOnRCStatusNotifications() out side of the for loop and call it only if disallowed_modules.size()>0

Reply:
@yang1070 Fixed 289cd19